hydroshare/local_settings.py

```diff
@@ -76,7 +76,7 @@
 
 
 # CartoCSS
-CARTO_HOME='/home/docker/node_modules/carto'
+CARTO_HOME='/node_modules/carto'
```

Comment:
@hyi - I could set it to something that is owned by **hydro-service**, like `/hs_tmp`, but not exposed to the host via the shared volume. It was initially `/home/docker`, which had this trait.